test/e2e-tests/README.md

```diff
@@ -0,0 +1,17 @@
+# AI Edge testing
+
+Place for testing and to verify the MLOps pipelines are working correctly. These tests use the `k8s.io/client-go` package to interact with the cluster. Using `oc login` to log into the cluster as normal should mean no setup is needed to run the tests.
+
+A locally install of the Go compiler is needed to run the tests. Go version `1.21` is required to run the tests.
+
+## Run tests locally
+```bash
+make test
+```
+Set the go binary used for testing that is in your `PATH`
+```bash
+make GO=go1.19 test
+```
+
+## CI/CD with Github Actions
+Not yet implemented
```

Comment:
I guess it's OpenShift CI rather than GitHub Actions now, so these lines may not be relevant any longer?

Reply:
This will probably be a dual ownership for both CI/CD workflows.  For github actions, we can execute of free workflows to linting and running some lightweight workflows in a [kind](https://kind.sigs.k8s.io/) cluster.  openshift-ci would be utilized for larger integration testing

Reply:
Ah right, understood, thanks :+1: